apps/web/vibes/soul/primitives/breadcrumbs/index.tsx

```diff
@@ -9,11 +10,23 @@ export interface Breadcrumb {
 }
 
 export interface BreadcrumbsProps {
-  breadcrumbs: Breadcrumb[];
+  breadcrumbs: Streamable<Breadcrumb[]>;
   className?: string;
 }
 
-export function Breadcrumbs({ breadcrumbs, className }: BreadcrumbsProps) {
+export function Breadcrumbs(props: BreadcrumbsProps) {
+  return (
+    <Stream fallback={<BreadcrumbsSkeleton />} value={props.breadcrumbs}>
+      {(breadcrumbs) => (
+        <BreadcrumbsResolved breadcrumbs={breadcrumbs} className={props.className} />
```

Comment:
I, personally, would remove BreadcrumbsResolved and just copy and paste code inside this Stream component.